source/spirv_target_env.h

```diff
@@ -28,6 +28,9 @@ bool spvIsOpenCLEnv(spv_target_env env);
 // Returns true if |env| is an OPENGL environment, false otherwise.
 bool spvIsOpenGLEnv(spv_target_env env);
 
+// Returns true if |env| is an implemented/valid environment, false otherwise.
```

Comment:
It's not that clear to me what it means for an environment to be "implemented/valid". I would assume that every value of the enum's range is valid in some sense (otherwise why would it be in the enum range?). Could you elaborate slightly in the comment?